content/en/05-reference/01-admin/09-user-mgmt.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "User Management & Sharing"
+metaTitle: "Reference | Admin | User Management & Sharing"
+metaDescription: "Share and manage access to your Pixie data."
+order: 6
+redirect_from:
+    - /admin/permissions/
+---
+
+<Alert variant="outlined" severity="info">This information applies to Pixie instances that use Auth0 for Authentication. <a href="https://docs.px.dev/installing-pixie/install-guides/community-cloud-for-pixie/">Community Cloud for Pixie</a> ships with Auth0. <a href="https://docs.px.dev/installing-pixie/install-guides/self-hosted-pixie/">Self-Hosted Pixie</a> must be <a href="https://docs.px.dev/reference/admin/authentication/#enabling-auth0">manually configured</a> to use Auth0. To learn how to invite users to Self-Hosted Pixie instances using open source authentication, see the Quick Start <a href="https://docs.px.dev/installing-pixie/install-guides/self-hosted-pixie/#invite-others-to-your-organization-(optional)">instructions</a>.
```

Comment:
it seems a little odd to me to have the invite instructions for non Auth0 OSS in the install instructions and the invite instructions for Auth0 OSS in this page. Can we put them in here in both cases, and also link from all sets of the install instructions to this page? I think we will want to say more about user management & sharing in non-auth0 over time as we add more features